gitops/snapshot.go

```diff
@@ -29,6 +29,9 @@ const (
 	// SnapshotTestScenarioLabel contains the name of the Snapshot test scenario.
 	SnapshotTestScenarioLabel = "test.appstudio.openshift.io/scenario"
 
+	// BuildPipelineRunPrefix contains the build pipeline run related labels and annotations
+	BuildPipelineRunPrefix = "build.appstudio"
```

Comment:
Can we have full FQDN here?
```suggestion
	BuildPipelineRunPrefix = "build.appstudio.openshift.io"
```

Reply:
It would be also compatible with style of this
`PipelinesAsCodePrefix = "pac.test.appstudio.openshift.io"`

Reply:
ah okay, I haven't noticed there are mutliple labels